quick-start-with-tidb.md

```diff
@@ -465,29 +466,27 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
     tiup cluster start <cluster-name>
     ```
 
-9. 访问集群：
+9. 访问集群端点：
 
-    - 安装 MySQL 客户端。如果已安装 MySQL 客户端则可跳过这一步骤。
+    - 安装 MySQL 客户端。如果已安装，则跳过这一步骤。
 
         {{< copyable "shell-regular" >}}
 
         ```shell
         yum -y install mysql
         ```
```

Comment:
This step is removed in the English doc https://github.com/pingcap/docs/pull/19960
https://github.com/pingcap/docs-cn/blob/e9a9a542dec02819ce4afcec5384c7455efc2926/quick-start-with-tidb.md?plain=1#L471-L477

Reply:
Added it back to the English version via https://github.com/pingcap/docs/pull/19960/commits/65e01ea5b880f559c4566863c29bbbefc2f3bbda.